config/filter.d/common.conf

```diff
@@ -44,14 +44,16 @@ __md5hex = (?:[\da-f]{2}:){15}[\da-f]{2}
 
 # bsdverbose is where syslogd is started with -v or -vv and results in <4.3> or
 # <auth.info> appearing before the host as per testcases/files/logs/bsd/*.
-__bsd_syslog_verbose = (<[^.]+\.[^.]+>)
+__bsd_syslog_verbose = <[^.]+\.[^.]+>
+
+__date_ambit = \[\]
 
 # Common line prefixes (beginnings) which could be used in filters
 #
 #      [bsdverbose]? [hostname] [vserver tag] daemon_id spaces
 #
 # This can be optional (for instance if we match named native log files)
-__prefix_line = \s*%(__bsd_syslog_verbose)s?\s*(?:%(__hostname)s )?(?:%(__kernel_prefix)s )?(?:@vserver_\S+ )?%(__daemon_combs_re)s?\s%(__daemon_extra_re)s?\s*
+__prefix_line = (?:%(__date_ambit)s)?\s*(?:(?:%(__bsd_syslog_verbose)s)\s*)?(?:(?:%(__hostname)s)\s*)?(?:(?:%(__kernel_prefix)s)\s*)?(?:(?:@vserver_\S+)\s*)?(?:(?:%(__daemon_combs_re)s)\s*)?(?:(?:%(__daemon_extra_re)s)\s*)?
```

Comment:
>  because as well as before and hereafter all the groups all separated with \s*

Are we looking at the same diff? ;) :
`(?:%(__hostname)s )?(?:%(__kernel_prefix)s )?` -> `?:(?:%(__hostname)s)\s*)?(?:(?:%(__kernel_prefix)s)\s*)?`
and others

Moreover, I would prefer to keep groupping (`(?:...)`) within the "variables" which then would make final regexp easier -- now it is difficult to match all the parantheses. Not to say that again, this goes away from "minimal change" -- what if someone has created a custom filter which relied on groups being present in those variables?